methcomp/mountain.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: utf-8 -*-
+
+"""Mountain plot.
+"""
+from typing import List, Union
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+
+from .comparer import Comparer
+
+
+class Mountain(Comparer):
+
+    """Mountain plot
+
+    A mountain plot is useful to compare how two methods perform on paired
+    measurements [krouwer_1995]. It forms the empirical cumulative distirbution
+    of the difference between pairs, folded at the median or any other quantile p
+    [monti_1995]. This makes it easy to see bias and spread. Furthermore, it has
+    been shown that the area under the curve AUC is the  mean absolute devation
+    from p [xue_2011].
+
+    Attributes
+    ----------
+    iqr : float
+        Interquartile range
+    n_percentiles : int
+        Number of percentiles to use - more gives a smoother mountain plot
+    result : Dict[str, Any]
+        Mountain calculation result with keys
+        * mountain - folded CDF in percent
+        * quantile - value along difference as function of quantile
+        * auc - area under curve
+        * median - quantile at median
+        * mountain_median - mountain at median
+        * iqr - interquantile range: index at low=50-iqr/2, high=50+iqr/2
+        * mountain_iqr - mountain low and high iqr
+
+    Examples
```

Comment:
Can you please also add a separate file in the examples/ directory? Just to keep it consistent for now. 

I still have to look into the Sphinx (documentation builder) API to see how we can effectively embed examples into the documentation.

Reply:
I'll keep this in this here and replicate it examples dir. I think it can belong in both places.